src/icons/add-clip-icon.tsx

```diff
@@ -11,7 +11,7 @@ const SvgAddClipIcon = (props: SVGProps<SVGSVGElement>) => (
     >
         <path
             fillRule="evenodd"
-            d="M15 0a1 1 0 0 1 1 1v3h3a1 1 0 1 1 0 2h-3v3a1 1 0 1 1-2 0V6h-3a1 1 0 1 1 0-2h3V1a1 1 0 0 1 1-1ZM1 4a1 1 0 0 0-1 1v4a1 1 0 0 0 2 0V6h3a1 1 0 0 0 0-2H1ZM0 19a1 1 0 0 0 1 1h4a1 1 0 1 0 0-2H2v-3a1 1 0 1 0-2 0v4Zm15 1h-4a1 1 0 1 1 0-2h3v-3a1 1 0 1 1 2 0v4a1 1 0 0 1-1 1Zm-7-5a3 3 0 1 0 0-6 3 3 0 0 0 0 6Z"
+            d="M15 0a1 1 0 0 1 1 1v3h3a1 1 0 1 1 0 2h-3v3a1 1 0 1 1-2 0V6h-3a1 1 0 1 1 0-2h3V1a1 1 0 0 1 1-1M1 4a1 1 0 0 0-1 1v4a1 1 0 0 0 2 0V6h3a1 1 0 0 0 0-2zM0 19a1 1 0 0 0 1 1h4a1 1 0 1 0 0-2H2v-3a1 1 0 1 0-2 0zm15 1h-4a1 1 0 1 1 0-2h3v-3a1 1 0 1 1 2 0v4a1 1 0 0 1-1 1m-7-5a3 3 0 1 0 0-6 3 3 0 0 0 0 6"
```

Comment:
Out of curiosity, where are these changes coming from, and what do they do to the svg?

Reply:
The new version of svgr is optimizing it by removing unnecessary svg codes.